dts/bindings/mbox/nordic,nrf-vevif-event-rx.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Nordic VEVIF (VPR Event Interface) - EVENT RX MODE
+
+  VEVIF provides support for inter-domain software signaling. It implements a set of events
+  intended for signaling within an interprocessor communication (IPC) framework.
+  When used in event rx mode, the VPR IRQ is used to receive events triggered by VPR core.
+
+  Example definition:
+
+    cpuppr_vpr: vpr@deadbeef {
+      ...
+      cpuflpr_vevif_event_rx: mailbox@0 {
+        compatible = "nordic,nrf-vevif-event-rx";
+        reg = <0x0 0x1000>;
+        interrupts = <76 NRF_DEFAULT_IRQ_PRIORITY>;
+        #mbox-cells = <1>;
+        nordic,events = <1>;
+        nordic,events-mask: <0x00008000>;
```

Comment:
wrong syntax, please fix

Reply:
The same error was in yaml for vevif_task, fixed.